### PR TITLE
azure free account module - closes #265

### LINKED ISSUE #265: understand options around Azure Free account

### PR DESCRIPTION
**Pull request by BenHesketh21**

### CODE REVIEW

File: topics/azure/modules/account-free-account/README.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+- [Exercises](#exercises)
+
+<!--TOC_END-->
+## Overview
+
+In this module we will go over Microsoft Azures option for a free account.
+
+## What is included?
+
+When you sign up for a free azure account, you have access to popular products like `virtual Machines`, `storage` and `databases` for the first 30 days. You then have access to these products for 12 months after you upgrade your account to pay-as-you-go pricing.
```
Comment (LukeBenson) [inline topics/azure/modules/account-free-account/README.md]:
```suggestion
When you sign up for a free Azure account, you have access to popular products like `virtual Machines`, `storage` and `databases` for the first 30 days. You then have access to these products for 12 months after you upgrade your account to pay-as-you-go pricing.
```

File: topics/azure/modules/account-free-account/README.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+As mentioned above, in the first 30 days you have access to popular Azure products.
+You also get `$150` credit to experiment with Azures services beyond the free ones available. 
+Additionally, there are more than `25` always free products including serverless, containers, and artificial intelligence. 
+You have access to these for the first 30 days and beyond, once you choose to `upgrade`.
+
+### No Automatic Charges
+
+As the name implies, this account is free. 
+You will not be charged unless you choose to upgrade.
+However, you can not exceed the `£150` credit with the services that not free to use.
```
Comment (LukeBenson) [inline topics/azure/modules/account-free-account/README.md]: Is it £ or $, you said $ previously

File: topics/azure/modules/account-free-account/README.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+You will be notified when the end of the 30 day free account is about to expire. 
+The option to upgrade to a paid subscription will be presented.
+If the account expires without being upgraded, any running Azure services being used by the account will be disabled.
+
+## Tutorial
+
+### Account Creation
+
+In this tutorial we will create a free azure account.
```
Comment (LukeBenson) [inline topics/azure/modules/account-free-account/README.md]:
```suggestion
In this tutorial we will create a free Azure account.
```

File: topics/azure/modules/account-free-account/README.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+Go [here](https://bit.ly/2T07whu) to start this tutorial.
+
+You then need to clock the `Start for free` button in the center of the page.
```
Comment (LukeBenson) [inline topics/azure/modules/account-free-account/README.md]:
```suggestion
You then need to click the `Start for free` button in the center of the page.
```

File: topics/azure/modules/account-free-account/README.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+You will then be asked to sign in or register a microsoft account.
+
+Next, you will be asked to fill in some more details, including an about you, verification by card, verification by phone and an agreement sections. 
+
+![register](https://qa-courseware-images.s3.eu-west-2.amazonaws.com/azure/account-free-account/001.png)
+
+In the agreement section you will need to tick the box that explains that you agree to the terms and conditions of the subscription. 
+
+And you have finished registering a free account for `Microsoft Azure!`
+
```
Comment (LukeBenson) [inline topics/azure/modules/account-free-account/README.md]:
```suggestion

Click the `Go to the portal` button to begin using your account.
```

File: topics/azure/modules/account-free-account/README.md [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+In this module we will go over Microsoft Azures option for a free account.
+
+## What is included?
+
+When you sign up for a free azure account, you have access to popular products like `virtual Machines`, `storage` and `databases` for the first 30 days. You then have access to these products for 12 months after you upgrade your account to pay-as-you-go pricing.
+
+### First 30 days
+
+As mentioned above, in the first 30 days you have access to popular Azure products.
+You also get `$150` credit to experiment with Azures services beyond the free ones available.
```
Comment (BenHesketh21) [inline topics/azure/modules/account-free-account/README.md]:
```suggestion
You also get `£150` credit to experiment with Azures services beyond the free ones available. 
```

**Pull request merged**